src/main/java/net/gensokyoreimagined/motoori/KosuzuParsesEverything.java

```diff
@@ -29,6 +29,7 @@
 public class KosuzuParsesEverything {
     private final ArrayList<Pattern> regexes = new ArrayList<>();
     private final Map<String, Map<UUID, Pattern>> placeholderRegexes = new HashMap<>();
+    private final ArrayList<String> syntaxBlacklist = new ArrayList<>();
```

Comment:
Fails to compile - if you assign here (since it's `final`), you cannot assign in the constructor. Remove the `= new ArrayList()`, and change the type to `List` by the return type of `getStringList`. In hindsight, `regexes` should also be of type `List`...